File: Arithmetic/P33.cpp
#include <iostream>

int gcd(int, int);

int main(){
    int num1, num2;

    std::cout << "Enter the numbers separeted by space : ";
    std::cin>> num1 >> num2;

    (gcd(num1, num2) == 1) ? std::cout << "Co-Prime Numbers" : std::cout << "Not a Co-Prime Numbers";

    return 0;
}

int gcd (int a, int b){
    return  (b == 0) ? a : gcd(b, a % b);
}

File: Python/P33.py
def gcd(a, b) :
    '''
    Gives GCD of the given two number.

    Uses recursion to find the GCD of two numbers.

    Parameters :
        a (int) : First Number.
        b (int) : Second Number.

    Returns :
        Integer which is GCD of given two numbers.
    '''

    if(b==0) : return  a #Base condition
    else : return gcd(b, a%b)

if __name__ == '__main__' :
    num1 = int(input("Enter first number : "))
    num2 = int(input("Enter Second number : "))

    # If GCD is 1 means number is co-prime.
    if gcd(num1, num2) == 1 :
        print("Given numbers ARE Co-Prime Numbers")
    else :
        print("Given Numbers are NOT Co-Prime Numbers")